### PR TITLE
Accessibility: Ensure alert close boxes describe what is being closed

### PR DESCRIPTION
**Pull request by npsp-reedestockton**
We corrected an issue with alert dialog close buttons that did not include information about what is being closed.
# Critical Changes

# Changes
 - To improve accessibility, we corrected a dialog box issue where close buttons didn't include information about what was being closed.

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by npsp-reedestockton** [comment]
Tracking [W-7965342](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008UEbiIAG/view)

**Comment by npsp-reedestockton** [comment]
@andrewyu-salesforce @RadSFDeveloper This is ready for review.
@sam-knox If you could take a look at the PR description, that would be great.

Thanks all!

File: src/components/UTIL_PageMessages.component [hunk 1]
```diff
@@ -38,8 +38,8 @@
                             <svg class="slds-button__icon slds-button__icon_large" aria-hidden="true">
                                 <path d="M14.6 11.9l6-6c.3-.3.3-.7 0-1l-.9-1c-.3-.3-.7-.3-1 0L12.6 10c-.1.2-.4.2-.6 0L6 3.9c-.3-.3-.7-.3-1 0l-1 .9c-.3.3-.3.7 0 1l6.1 6.1c.1.1.1.4 0 .6L4 18.6c-.3.3-.3.7 0 1l.9.9c.3.3.7.3 1 0l6.1-6c.2-.2.5-.2.6 0l6.1 6c.3.3.7.3 1 0l.9-.9c.3-.3.3-.7 0-1l-6-6c-.2-.2-.2-.5 0-.7z"/>
                             </svg>
-                            <span class="slds-assistive-text">Close</span>
-                        </button> 
+                            <span class="slds-assistive-text">{!m.severityLabel}: {!m.summary} {!$Label.bdiBtnClose}</span>
```
Comment (RadSFDeveloper) [inline src/components/UTIL_PageMessages.component:41]: Can we use a getter method to return assistive text? Also, the ":" is not recommended to be used outside of the custom labels for the translation requirements.

**Review by sam-knox — APPROVED** [review]

File: src/components/UTIL_PageMessages.component (hunk 1 above)
Comment (npsp-reedestockton) [reply src/components/UTIL_PageMessages.component]: @RadSFDeveloper Requested changes have been made.  Thanks!!

**Review by RadSFDeveloper — APPROVED** [review]
Looks good, thanks @npsp-reedestockton!

**Review by npsp-reedestockton — COMMENTED** [review]
@RadSFDeveloper @andrewyu-salesforce The changes made to this code are significant enough that I would appreciate another review.  Thanks!!

**Review by RadSFDeveloper — APPROVED** [review]
It looks good, thanks Reede!

**Pull request merged**